.github/workflows/jekyll-gh-pages-avatar.yml

```diff
@@ -34,18 +34,18 @@ jobs:
       - name: Clone autils repo
         uses: actions/checkout@v3
         with:
-          repository: avocado-framework/autils
+          repository: avocado-framework/aautils
           path: resources/autils
           ref: main
 
       - name: List files in autils directory
-        run: ls -R resources/autils/metadata/autils/
+        run: ls -R resources/autils/metadata/archive/
```

Comment:
IMO this should be only `resources/autils/metadata/` because the `archive` dir is related only to the archive utilities other utility types will be in different directories, network for example.

Reply:
Hi Jan, 

Thanks for explaining this i've sorted it in the latest push :)